pygus/gus/models/config.py

```diff
@@ -46,6 +46,7 @@ def __init__(
         weather: Union[Dict, WeatherConfig],
         tree_density_per_ha: int = None,
         project_site_type: str = "park",
+        light_loss_multiplier: float = 0.1
```

Comment:
@bulentozel I'm wondering if the default value should be 0.75, assuming the previous behaviour of "Urban"?

Alternatively, we could make a base class called GUSModel or something, and create sub-classes Urban and Miyawaki, that differ only by configurations of certain site parameters. These could continue to be modified explicitly in the SiteConfig.
What do you think?